types/fxjs/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for fxjs
```

Comment:
Could you add a version number (for example, `// Type definitions for fxjs 0.15`) to the header? I think if you do that, you don't need to disable `dt-header` in your tslint.json.

Reply:
dt-header option changed true